layouts/_default/_markup/render-link.html

```diff
@@ -1,16 +1,16 @@
-{{$trimmed := strings.TrimSuffix ".md" (.Destination | safeURL)}}
-{{$dashedurl := replace $trimmed "%20" "-" }}
+{{$dashedurl := replace .Destination "%20" "-" }}
 {{$external := strings.HasPrefix $dashedurl "http" }}
 {{- if $external -}}
-<a href="{{ $dashedurl }}" rel="noopener">{{ .Text | safeHTML }}</a>
+	<a href="{{ $dashedurl }}" rel="noopener">{{ .Text | safeHTML }}</a>
 {{- else -}}
-{{$spacedurl := replace $trimmed "%20" " " }}
-{{$fixedUrl := (cond (hasPrefix $spacedurl "/") $spacedurl (print "/" $spacedurl)) | urlize}}
-{{$nonexistent := eq (.Page.GetPage $spacedurl).RelPermalink ""}}
-{{$rooted := default $spacedurl ((.Page.GetPage $spacedurl).RelPermalink) }}
-<a
-  {{if not $nonexistent}}href="{{$rooted}}"{{end}}
-  rel="noopener" class="internal-link{{if $nonexistent}} broken{{end}}"
-  data-src="{{$rooted}}">{{- .Text | safeHTML -}}
-</a>
+	{{$trimmed := strings.TrimSuffix ".md" (.Destination | safeURL)}}
+	{{$spacedurl := replace $trimmed "%20" " " }}
+	{{$fixedUrl := (cond (hasPrefix $spacedurl "/") $spacedurl (print "/" $spacedurl)) | urlize}}
+	{{$nonexistent := eq (.Page.GetPage $spacedurl).RelPermalink ""}}
+	{{$rooted := default $spacedurl ((.Page.GetPage $spacedurl).RelPermalink) }}
+	<a
```

Comment:
formatting in this file gets preserved when it renders so would be nice to unindent this block

Reply:
My bad. done